### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by Flashguru**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- I put my info and picture in it.

### CODE REVIEW

**Comment by abishekvashok** [comment]
@Flashguru do you know to resolve merge conflicts if yes please resolve them or we will do it for you.

File: _data/students.yml [hunk 1]
```diff
@@ -835,3 +835,11 @@
   bio: Decent geek, Future web developer
   lat: 28.5497020
   lng: 77.3523170
+  
+- name: Neil Bedagkar
+  github: flashguru
+  twitter: @flamingmangoe
```
Comment (abishekvashok) [inline _data/students.yml]: Please don't include the @ character and there are merge conflicts we could resolve them but you should remove the @ character

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: _data/students.yml (hunk 1 above)
Comment (Flashguru) [reply _data/students.yml]: I don't know how to resolve merge conflicts but I will remove the @. thanks

**Comment by robbyoconnor** [comment]
I resolved the conflict.

**Comment by robbyoconnor** [comment]
@Flashguru in the future title your pull requests better.

**Pull request merged**